### DISCUSSION TITLE
Setting up - monitors_mute_state?

### DISCUSSION THREAD

**Discussion by ctankep23** [matt8707/hass-config#60] [Q&A]
Hey! I followed the guide to setup your config but am unable to restart HA due to: "Error loading /config/configuration.yaml: Secret monitors_mute_state not defined"

I am a totalt newb. What is wrong?

**Comment by matt8707** [comment]
Please read it again. `include/binary_sensor.yaml` is not included in the guide

https://github.com/matt8707/hass-config/blob/master/INSTALL.md
